Samples/AppLifecycle/Instancing/cpp-winui-packaged/CppWinUiDesktopInstancing (Package)/Package.appxmanifest

```diff
@@ -7,13 +7,13 @@
   IgnorableNamespaces="uap rescap">
 
   <Identity
-    Name="c4a3b17f-0039-493a-a7b5-a82222ffe041"
-    Publisher="CN=Microsoft Corporation"
+    Name="316a7f97-7cd3-4657-87b2-fd866c21e546"
+    Publisher="CN=dahoehna"
```

Comment:
I don't think you want your email alias in the repo. @btueffers what is the guidance here? should all samples have CN=Microsoft or something like CN=WinAppSDKSamples?

Reply:
I think this file auto-updates when making a cert for publishing.  I did change it to Microsoft.